templates/srv/www/sites/composer.json.j2

```diff
@@ -0,0 +1,2 @@
+{% set _composer_json = (roundcube__register_composer_json.stdout | from_json) %}
+{{ _composer_json | to_nice_json }}
```

Comment:
Why is this needed? It seems to be a roundabout way to copy the contents of the `dist` file to another file right next to it. Won't simple `cp` or even fancy `copy` module with `remote_src: True` do the trick?